tests/conftest.py

```diff
@@ -1,16 +0,0 @@
-import nkms.db
```

Comment:
Didn't quite get about the removal of this file. It sets up the debug db which we're using to store kFrags (if not doing that, it'll reuse the per-user db which is bad).